web/views.py

```
@@ -1242,6 +1244,49 @@
+
+def crowdfunding_list(request):
+    """
+    Display all active crowdfunding campaigns.
+    Active campaigns are those that are approved and live.
+    """
+    campaigns = Campaign.objects.filter(approved=True, live=True).order_by("-created_at")
+    return render(request, "crowdfunding_list.html", {"campaigns": campaigns})
+
+@login_required
+@teacher_required
+def crowdfunding_create(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `crowdfunding_create`

(ANN201)